### ISSUE TITLE
OpenStack Installer fails on Ubuntu 16.04.3 LTS with WORKLOAD_ERROR_STATES,  IndexError: list index out of range

### ISSUE THREAD

**Issue by w4tdot** [conjure-up/conjure-up#1306]
Hello,

First of all I am doing everything on a clean install of Ubuntu 16.04.3 LTS.

- [NO ] Is this problem already documented at https://docs.ubuntu.com/conjure-up/en/troubleshoot#common-spell-problems
- [NO] Is conjure-up running inside a virtual machine?
- [YES] Is this problem reproducible with `sudo snap refresh conjure-up --edge`?

## Please provide the output of the following commands

```
which juju
/snap/bin/juju
juju version
2.3.1-xenial-amd64


which conjure-up
/snap/bin/conjure-up

conjure-up --version
conjure-up 2.5.2

which lxc
/snap/bin/lxc config show
/snap/bin/lxc version
/snap/bin/lxc

/snap/bin/lxc config show
config:
  core.https_address: '[::]'

/snap/bin/lxc version
2.21

cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.3 LTS"
```
```
tar cvzf conjure-up.tar.gz ~/.cache/conjure-up - OK
[conjure-up.tar.gz](https://github.com/conjure-up/conjure-up/files/1639282/conjure-up.tar.gz)

```

## Sosreport

Please attach a sosreport:
OK
[sosreport.txt](https://github.com/conjure-up/conjure-up/files/1639290/sosreport.txt)

```
sudo apt install sosreport
sosreport
```
The resulting output file can be attached to this issue. OK

## What Spell was Selected?
OpenStack

## What provider (aws, maas, localhost, etc)?
Localhost.

### MAAS Users
Which version of MAAS? I dont use MAAS.

## Commands ran

Please outline what commands were run to install and execute conjure-up:
https://www.ubuntu.com/download/cloud/try-openstack
1. Install clean Ubuntu.
2. sudo apt update
3. sudo apt upgrade -y
4. sudo snap install conjure-up --classic
5. Login again (because else I get error on lxd not found)
6. conjure-up

## Additional Information
I was following what was stated on Ubuntu's webpage. It states it takes just one hour. Now I have spent several hours with it and a **** "IndexError: list index out of range" break everthing. Seriously...

For  further references I attach screenshots.
![screenshot from 2018-01-17 14-06-19](https://user-images.githubusercontent.com/35531914/35047724-17719052-fb9b-11e7-9449-63871a1acc63.png)
![screenshot from 2018-01-17 14-08-29](https://user-images.githubusercontent.com/35531914/35047725-17899fe4-fb9b-11e7-971f-a141f42cf40a.png)
![screenshot from 2018-01-17 14-09-26](https://user-images.githubusercontent.com/35531914/35047726-17a0c7dc-fb9b-11e7-89b7-4cbae2ee10f6.png)
![screenshot from 2018-01-17 14-12-43](https://user-images.githubusercontent.com/35531914/35047727-17b86e32-fb9b-11e7-8121-d54774a2ed20.png)
![screenshot from 2018-01-17 14-15-07](https://user-images.githubusercontent.com/35531914/35047728-17cf7cda-fb9b-11e7-8b1b-1acda0f5c078.png)
![screenshot from 2018-01-17 14-16-16](https://user-images.githubusercontent.com/35531914/35047729-18106042-fb9b-11e7-8126-670ab318c19e.png)
![screenshot from 2018-01-17 14-18-26](https://user-images.githubusercontent.com/35531914/35047731-18c8a0e4-fb9b-11e7-9663-0cce2aebe9d5.png)
![screenshot from 2018-01-17 14-24-48](https://user-images.githubusercontent.com/35531914/35047732-18dfecf4-fb9b-11e7-90f4-11aee73e08e5.png)
![screenshot from 2018-01-17 14-32-53](https://user-images.githubusercontent.com/35531914/35047733-192e171c-fb9b-11e7-871f-58fbf545b899.png)
![screenshot from 2018-01-17 14-37-53](https://user-images.githubusercontent.com/35531914/35047734-1945cc68-fb9b-11e7-8bef-3f352bb66dec.png)
![screenshot from 2018-01-17 14-44-06](https://user-images.githubusercontent.com/35531914/35047736-196acad6-fb9b-11e7-8fde-5d6912878e3d.png)
![screenshot from 2018-01-17 14-48-50](https://user-images.githubusercontent.com/35531914/35047737-1982c9ce-fb9b-11e7-98ad-20500bd8853d.png)
![screenshot from 2018-01-17 15-03-14](https://user-images.githubusercontent.com/35531914/35047738-19de88d6-fb9b-11e7-8aeb-9f00374c7fb8.png)
![screenshot from 2018-01-17 15-08-36](https://user-images.githubusercontent.com/35531914/35047739-19f8762e-fb9b-11e7-9953-f4a1798208ca.png)

**Comment by w4tdot** [comment]
After restart I got the following when choosing "OpenStack with NovaLXD":

![screenshot from 2018-01-17 15-51-34](https://user-images.githubusercontent.com/35531914/35048810-6a43afec-fb9e-11e7-9d35-eb2b451d63e6.png)

**Comment by w4tdot** [comment]
I have solved the issue: apt-purge conjure-up
Then installed OpenStack manually.

**Comment by lyndon160** [comment]
That is not a solution!

**Comment by johnsca** [comment]
This was fixed with #1327 but this bug was never updated to indicate that.

**Comment by johnsca** [comment]
@lyndon160 How did you come across this bug?  Are you encountering something similar?
